adoc/extensions/sycl_khr_default_context.adoc

```diff
@@ -0,0 +1,57 @@
+[[sec:khr-default-context]]
+= SYCL_KHR_DEFAULT_CONTEXT
```

Comment:
Yes, we intentionally did not add the `ext_` prefix.

Section 6.2 reserves the vendor strings "khr" and "KHR" in order to give the WG the option to define official extensions this way, but the WG decided not to do this because "ext_khr" is more verbose than simply "khr".